docs/.vuepress/components/TwoColumnGrid.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="two-column__wrap">
+    <cdr-grid class="two-column__grid">
+      <div
+        v-for="(example, idx) in examples"
+        :key="`${example.type}${idx}`"
+      >
+        <figure class="two-column">
+          <cdr-img
+            class="two-column__image"
+            :src="$withBase(`/${example.image}`)"
+            :ratio="getRatio(example.ratio) || '16-9'"
+            :alt="example.alt || ''" />
+          <figcaption class="two-column__caption">
+            <span>{{ example.caption }}</span>
+            <slot/>
+          </figcaption>
+        </figure>
+      </div>
+    </cdr-grid>
+  </div>
+</template>
+
+<script>
+import Vue from '$vue';
```

Comment:
Don't think you need this import, it brings in the full build of vue ... runtime compiler and alll

Reply:
Yup removed it and everything works. Removed the import from the `<DoDont>` component I used as a template as well.